avocado/plugins/multiplex.py

```diff
@@ -35,23 +34,28 @@ def __init__(self, *args, **kwargs):
 
     def configure(self, parser):
         parser = super(Multiplex, self).configure(parser)
+        parser.add_argument("--summary", type=int, help="Verbosity of "
```

Comment:
well since the use of warn, error and critical would be probably not applicable here, this gives us the range of 20 with just 2 human friendly names. Tot his I'd prefer custom enum defined as "brief,normal,verbose,full". Still the numbers make more sense to me as we are dealing with different plugins and even our plugin actually accepts bitmask nowadays...